MarsQA-1/Features/LanguageFeature.feature

```diff
@@ -0,0 +1,67 @@
+﻿Feature: Language
+
+As a user, I need to manage my language details in my profile
+so that I can perform actions such as add, edit, or delete my language records
+
+@Lang
+Scenario Outline:1 Add Languages and language level for user profile
+	Given When i Login to the Application and navigates to language section
+	When i add <language> and <languagelevel> 
+	Then <language> and <languagelevel> should be added succesfully
+
+	Examples: 
+	| language  | languagelevel |
+	| 'Eng'| 'Fluent'|  
+	| 'Ping'| 'Fluent'|  
+	| 'Sing'| 'Fluent'|  
+
+	Scenario Outline: 2 Edit Languages and Language level
+	Given   User is logged in and navigates to profile 
+	When    User edits the record with <newlanguage> and <newlanguagelevel> 
+	Then    profile gets updated with modified <newlanguage> and <newlanguagelevel> values
+
+	Examples: 
+	| newlanguage | newlanguagelevel |
+	| 'Hindi' | 'Basic' |
+
+	Scenario Outline: 3 Delete Languages from User Profile
+	Given   User is logged in and navigates to delete section
+	When    User deletes <language> from profile
+	Then    <language> Should be deleted
+
+	Examples: 
+	| language |
+	| 'Hindi'  |
+	
+	Scenario Outline: 4 Add Language record with invalid data
+	Given  User is logged in and navigates to profile
+	When User tries to add new record with invalid data <language> and <languagelevel>
+	Then language record should not be added
+	Examples: 
+	| language | languagelevel |
+	|    ''      |  'Conversational' |
+	|    'French'      |    ''          |
+	|    ''      |    ''          |
+
+	Scenario Outline: 5 Add Language record with already existing data
+	Given  User is logged in and navigates to profile
+	When User tries to add new record with already existing data <language> and <languagelevel>
+	Then  user should not be able to add record
+	Examples: 
+	| language | languagelevel |
+	| 'Ping'| 'Fluent'| 
+	
+
+
+
+	Scenario: 6 Edit Language record without changing data
```

Comment:
you are missing 

Please increase your coverage around the features you  are testing here is an idea of what is missing 

• Negative testing with valid input
• Negative testing with invalid input
• Destructive testing

E.g of above scenarios
Happy path tests check basic functionality and the acceptance criteria of the API. We later extend positive tests to include optional parameters and extra functionality.

The next group of tests is negative testing where we expect the application to gracefully handle problem scenarios with both valid user input (for example, trying to add an existing username) and invalid user input (trying to add a username which is null).

Destructive testing is a deeper form of negative testing where we intentionally attempt to break the API to check its robustness (for example, sending a huge payload body in an attempt to overflow the system).

Reply:
Have made changes as Suggested